### ISSUE TITLE
Режим "lite pk" обсуждение

### ISSUE THREAD

**Issue by joopel** [bylins/mud#606]
Сначала пара слов для чего режим: режим нужен чтоб снизить цену ошибки в военных действиях, сейчас видно как на игроков, особенно мутантов давит стаф и потеря экспы, не происходит войны какой она должна быть, если одна сторона слила 1-2 боя у нее почти нет шансов реабилитироваться, если конечно у ее лидеров не яица с железный кулак и куча свободного времени. Также есть момент конфликтов между игроками, у них нет задачи уничтожить друг друга, но помериться писюнами охота и чтоб для кланов их забавы не стали накладны, в виде потери стафа.

**1) Режим включается командой режим лайтпк**. Самое включение/выключение происходит через час после ввода команды, переключаться между режимами можно 1 раз в месяц. Доступен с 21 морта, когда есть уже что терять. игроки доступны по команде кто лайтпк. Сейчас название не принципиально. На время активации и деактивации режима игрок не может шевелиться/вступать в группы, что проще так и делаем.

**2) ПК игроков с включенным режимом** - при убийстве остаются останки как с арены, в них попадает рандом 3-4 вещи с убитого, теряется 1/6 опыта. Убитый появляется на ренте как после боя на арене, без этих вещей. На убийцу вешается такой же флаг боевые действия. Если происходит группове пк, происходит подсчет игроков с включенным режимом внутри группы, если 2/3 из них имеют включенный режим, считается что у всех членов группы он включен.

**3) ПК игроков если у одного из них НЕ включен режим** - не важно у кого он включен и выключен, происходит обычное пк, тоже самое распространяется и на группу.

**_Тем самым, игроки которые не подходит такой режим, просто его не включают._**

Какие вижу плюсы и минусы от режима: все войны уверен будут вестись в этом режиме, так как всем лидерам хочется снизить цену за свои ошибки, войны станут динамичней и будут длиться месяцы, а не годы.

Из минусов вижу что скорее всего игроки будут поголовно бегать с этим режимом, надеясь что у пк он будет тоже включен и скорее всего так и будет, в итоге мы придем к тому что все бегают в режиме лайтпк. Плохо это или хорошо не знаю, скорее всего хорошо так как лучше потерять 3-4 стафины чем весь стаф и афкать/уходить с мада, как мы наблюдаем в последнее время.

Хочу увидеть аргументы против. Аргумент зачем что-то менять не принимается, я вижу боль игроков и потребность в этом режиме.

**Comment by lrdx** [comment]
А в чем преимущество режим фулпк получается?

**Comment by joopel** [comment]
> А в чем преимущество режим фулпк получается?

фул пк, как сейчас остается в _не изменном виде_! Это вопрос с подвохом какой-то? Преимущества фулл пк очевидны

**Comment by LdanDSS** [comment]
Вопрос даже не в самом режиме ПК, а в принципе в системе войн. Сейчас война это абсолютно бесцельная фигня, результат которой часто понятен до ее начала и все скатывается в систему "кто кого пересидит". 

Кроме лайт-пк надо подумать о **целях** войны и ее системе.

Например можно сделать систему княжеств, где каждая дружина имеет список территорий, которыми владеет. Владение территорией дает, например, + к экспе, + к статам или дает постоянный определенный количеством подконтрольных территорий доход кун. 

При объявлении войны целью является 1) Захват территорий 2) Успешная атака вражеского замка и вот здесь можно применить систему фулл-пк/лайт-пк. 

**В лайт-пк:**
Цель войны объявляется атакующей стороной (та которая объявила войну), но защищающаяся сторона выбирает место (определенную, попавшую под удар территорию) и время боя. Т.е. в назначенное время атакующая сторона должна выполнить какую-нибудь задача (убить, например, какого-нибудь важного моба/группу мобов в городе, а защищающаяся дружина должна этому помешать).  И так , напимер, пока защитники не отобьются , либо атакующая сторона не захватит определенное количество территорий. Тут можно отбалансить так, чтобы захватить надо было для успеха войны определенное количество территорий. 

**В фул-пк.**
Атакующая сторона может сразу забить на захват территорий и атаковать вражеский замок. Защитники также назначают время штурма. В замке дружине помогают мобы, итп. В общем, это можно отбалансить. Если атакующие выигрывают то получает весь стаф с трупов, + часть стафа с храна и кун как виру, а также на выбор подконтрольную проигравшим территорию и война заканчивается. Очевидно, что это "ва банк" для атакующих. При провале они теряют весь стафф и тоже часть территорий в пользу защитников. 

Для того, чтобы войны назначались редко но метко, указать, что между войнами одних и тех же дружин должно пройти не менее 3-х месяцев ИРЛ, чтобы дать возможность проигравшим откачаться и взять реванш, если они пожелают, или имели хороший шанс отбиться от новой атаки.

**Comment by Nimrok** [comment]
Всегда был за то, чтобы задать войнам рамки. Спустя время или еще по каким-то критериям, но именно насильно заканчивать войну. Да победитель не сможет уничтожить врага физически, морально и виртуально, но все равно остается победителем с захваченным имуществом. Позволить проигравшим какое-то время быть полностью защищенными от победителей и вот уже есть стимул играть после поражения. В это время можно даже подрезать какие то ресурсы у победителя и при этом быть безнаказанными, но спустя пару недель или месяц все равно все вернется на круги своя и победитель снова докажет что он крут, или же не сможет этого сделать и вот уже неплохая система камбеков.

**Comment by Nimrok** [comment]
Такой вариант будет приближен к средневековым войнам европы, где аристократия не старалась уничтожить друг друга, а лишь делила зоны влияния. 
А по поводу захвата территорий - если организовать эту систему так, чтобы в принципе нельзя было одному клану держать все территории и снова возвыситься запредельно. Чтобы просто никому не хватило ресурсов получить Все. Но при этом всем дать возможность и побороться друг с другом и позахватывать не очень конкурентные участки. Одни территории очень ценны и за них можно подраться, другие просто посредственно цены и к кому ближе те и берут.

**Comment by Svjatovit** [comment]
Давайте не оффтопить про войны? Тема про режим, его и обсуждайте. Я, если что, идею поддерживаю, хотя надо подумать и пошлифовать.

**Comment by ReynirOddmarson** [comment]
Я тоже поддерживаю эту идею, это всяко оживит мир.
Так же я бы ввел для игроков режим заклятый враг - пример : 
не нравится тебе персонаж N, да и ты ему не слишком, ты включаешь этот режим для него и если он тоже подтверждает, то вы можете убивать друг-друга на протяжении определенного временного промежутка и к вам в ваши мини разборки никто не может влезть.

**Comment by Svjatovit** [comment]
"Заклятый враг" абузоопасно. Что значит "не могут влезть"? Не могут вообще атаковать кого-то из парочки? Абьюзы перечислить или видно так?

**Comment by ReynirOddmarson** [comment]
да, и парочка кроме друг-друга тоже не может никого атаковать, а главное если никто из них никого не убьет за действие этого флага, то обоим вешается весомый штраф. так менее абьюзно?
